1-exercises/C-array-every/exercise.js

```diff
@@ -5,7 +5,7 @@
 let students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 let group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-let groupIsOnlyStudents; // complete this statement
+let groupIsOnlyStudents = group.every((i) => students.includes(i));
```

Comment:
This is good but I would use a more descriptive name, e.g.
```suggestion
let groupIsOnlyStudents = group.every((name) => students.includes(name));
```